js/src/dashboard/summary-section/index.js

```diff
@@ -54,7 +56,11 @@ const FreePerformanceCard = () => {
 };
 
 const PaidPerformanceCard = () => {
-	const { formatAmount } = useCurrencyFactory();
+	// Spend amount is given in the Ads' currency, but Total Sales is in store's currency.
```

Comment:
Interesting. Thanks, for pointing that :pray: 
I wonder what is the best place to add those references. As that would be a bit longer comment, so it's better not to duplicate it in many places. Do you have some idea?

----

Sorry, my brain is fried today, I'll take another look tomorrow, but do I get it correctly, that theoretically, we could have even more currencies?

> Conversion values will always be reported to you in your Google Ads billing currency. However, you can assign a different currency to your Google Analytics conversions if those are measured in a currency different from your billing currency. Google Ads will import the Google Analytics conversions in that designated currency and then convert it for you to your billing currency.

- Billing
- Current store's  currency
- Store's currency, at a time when the conversion was recorded

Reply:
> I wonder what is the best place to add those references. As that would be a bit longer comment, so it's better not to duplicate it in many places. Do you have some idea?

I'd probably create a README.md at the *js/src/report* directory for it, and point all relevant usages to it with a simple comment for each.

---

> do I get it correctly, that theoretically, we could have even more currencies?

I read the Google description again, and I may have misunderstood it yesterday.

I think the "Google Ads billing currency" should be equal to the "Google Ads currency" in our code. When tracking the conversion value, if the currency specified in the tracking data is different from the Google Ads billing currency, it will be converted to the billing currency. If no currency is specified when sending the tracking, the conversion value is recorded directly without the exchange rate converting.

In addition, there may be different checkout (order) currencies on the merchant site, such as multi-currency checkout, or if the merchant changes their store currency.

On the premise that Google converts exchange rates to billing currencies. If the report data of conversion value is to be displayed in a single column in each row, it could be inferred that the currency of the conversion value is the same as the Google Ads billing currency. Otherwise, when more than one currency is recorded during tracking, it is impossible to present each currency's report by the current report data structure.

Therefore, I assume the currency of "Total Sales" should be displayed in the same as the Google ads' currency. However, I have no real data to confirm this hypothesis, nor can I find the relevant explanations from available code or documentation. From Google's Merchant Center and Ads sites, the conversion value does not seem to be marked with a currency. So, maybe we could also consider not showing its currency for now until we confirm that with Google team.

cc @jconroy

Reply:
🚧 
@tomalec, we got the reply from Google team that the currency of conversion value in our case should be the same as Google Ads currency. So it needs some adjustments for the "Total Sales".

Reply:
Fixed by https://github.com/woocommerce/google-listings-and-ads/pull/1020/commits/e66403e3e092462df913b08f3e8d8527a39692aa.